src/main/java/uk/gov/companieshouse/company_appointments/config/Config.java

```diff
@@ -61,4 +67,12 @@ public Supplier<InternalApiClient> internalApiClientSupplier(
             return internalApiClient;
         };
     }
+
+    @Bean
+    public ObjectMapper objectMapper() {
+        return new ObjectMapper()
+                .registerModule(new JavaTimeModule())
+                .setDateFormat(new SimpleDateFormat("yyyy-MM-dd"))
```

Comment:
First problem was, it couldn't deserialise LocalDate objects without adding that `new JavaTimeModule`. 

Once i added that, one test was failing where it was deserialising a Mongo iso date into an array of integers in the response body on a GET request (e.g. the GET response was showing `"appointed_on": [2023, 10, 05]`). Adding the `setDateFormat` part fixed this and all tests pass.

Reply:
Can you see any issues with this approach?

Reply:
What about fields like delta_at and updated.at? Do they get deserialised and persisted okay? It would be good to test the get endpoints with these changes too to make sure we can access the appointment persisted.

Reply:
Ok, will look into this. I think another option is to add a `@JsonFormat` annotation to the response model in `private-api-sdk-java` and set the expected format on the `appointed_on` field.

Reply:
Managed to get Tilt/Docker working(ish). The only GET I can't get working is `/company/{company_number}/officers` but the rest seem to be working fine with the `appointed_on` field. 

Re the delta_at and updated.at, these also seem fine, being persisted as a Mongo Date object as per usual. The they are also persisting with the date provided in an officers delta correctly - e.g. `"delta_at": "20231009101003950844"` is being persisted as `2023-10-09T10:10:03.950Z`.